README.md

```diff
@@ -6,6 +6,14 @@
 
 `npm run node-hack`
 
+### Podfile install for iOS
+
```

Comment:
What are podfiles?

Reply:
It's an iOS package manager. Sentry needs to add some native codes, so we must install an iOS native lib of Sentry.

Actually Apple already released a new package manager 'Swift Package Manager' but it's not supported for React native yet

Reply:
Thanks! I think you can add this information at the beginning of this section.

Reply:
Done